src/cromshell/metadata/command.py

```diff
@@ -79,7 +79,7 @@ def get_workflow_metadata(
     api_workflow_id: str,
     timeout: int,
     verify_certs: bool,
-) -> str:
+) -> dict:
     """Uses requests to get the metadata or sub-metadata of a workflow
```

Comment:
This change was more of a correction, I was using the function in the ESC command and Pycharm was throwing a warning because the variable type didn't match up. The fix shouldn't affect other things. This function only gets called in the metadata command at line 113, the output gets passed to pretty_print_json at line 120 which accepts strings or dictionaries.  
I'll also update the description